content/foundations/color.mdx

```diff
@@ -236,6 +237,30 @@ Not all colors pair well with each other. There are combinations of backgrounds
   src="https://user-images.githubusercontent.com/378023/156498652-b1121c86-a670-4a92-9a8d-9db3c7e0ed9a.png"
 />
 
+## Accessibility
+When designing with colors consider the following to make your UI visually accessible.
+
+### Assure adequate color contrast
+Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 
```

Comment:
As the [functional role pairing](https://primer.style/design/foundations/color#combining-colors) guarantees WCAG compliance already, should we put part of this content within the "Combining colors" section? For instance, describing what an accessible combination means and reinforcing the usage of the functional pairing. Maybe the images with combinations can be improved by adding the contrast ratio to go hand in hand with the content.

Reply:
👍 

To the mention below, I'd use "functional colors" or "functional role pairing" instead of "primer colors"

Reply:
Good one, changed that. Thanks.